### ISSUE TITLE
Add functions to get working and user directory.

### ISSUE THREAD

**Issue by frankenjoe** [audeering/audeer#156]
In #155 we implement a function that returns the directory of the current script. It might be a good fit to also add a function that returns the current working directory (e.g. `working_dir()`) and user directory (e.g. `user_dir()`).

**Comment by frankenjoe** [comment]
No, `audeer.working_dir()` would return `os.getcwd()` and `audeer.user_dir()` return `audeer.path("~")`. Then we provide functions for the three special directories. But maybe that is not really needed.

**Comment by hagenw** [comment]
It is not really needed as there exist alternatives that involve only a single function call (as was not the case with `audeer.script_dir()`).
But I see the point for providing these functions for symmetric reasons ;)

`audeer.user_dir()` has the problem, that it is not completely obvious what is meant. As I first thought you are referring to the directory in which the user was when starting a script. So maybe `audeer.home_dir()` would be a better choice?

**Comment by frankenjoe** [comment]
Right, if we decided to add these functions then `audeer.home_dir()` would be a better fit.

**Comment by frankenjoe** [comment]
> which the user was when starting a script.

Mhh, if it's possible to get this location, this would also be a nice function to have :)
